src/components/play-mode/chat/Chat.jsx

```diff
@@ -37,6 +37,9 @@ function ChatInput () {
 
         const handleActiveKey = ( event ) => {
 
+            const inputFocused = document.activeElement && ['INPUT', 'TEXTAREA'].includes( document.activeElement.nodeName );

```

Comment:
I think there is a util for this now.

Reply:
https://github.com/webaverse/app/blob/07f4cfb4407d70a8bf49b622c2621844040a099f/game.js#L1241